pkg/fanal/analyzer/language/nodejs/yarn/yarn_test.go

```diff
@@ -357,17 +402,25 @@ func Test_yarnLibraryAnalyzer_Analyze(t *testing.T) {
 			name: "package uses `latest` version",
 			dir:  "testdata/latest-version",
 			want: &analyzer.AnalysisResult{
-				Applications: []types.Application{
+				Applications: []ftypes.Application{
 					{
-						Type:     types.Yarn,
+						Type:     ftypes.Yarn,
 						FilePath: "yarn.lock",
-						Packages: types.Packages{
+						Packages: ftypes.Packages{
+							{
+								ID:           "4b1aec5d292e8d22",
```

Comment:
Perhaps we want to add some prefix for hash.

We also will need to use hash for [cargo](https://github.com/aquasecurity/trivy/issues/8013). 
So it can be one prefix for all type, or we can relate prefix with type.